modules/express/src/fetchEncryptedPrivKeys.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * This tool will help you creates a private key JSON file to be used with the external singing mode feature.
+ * It creates a JSON file containing a list of wallet IDs and their corresponding encrypted user private keys.
+ *
+ * To run this file, use the command:
+ * `yarn ts-node <path/to/fetchEncryptedPrivKeys.ts>`
+ *
+ * Copyright 2022, BitGo, Inc.  All Rights Reserved.
+ */
+import { writeFile } from 'fs';
+import { BitGo } from 'bitgo';
+
+type Output = {
+  [key: string]: string
+}
+
+type WalletIds = {
+  [key: string]: string[]
+}
```

Comment:
added this `WalletIds` type here so that the compiler wouldn't complain about not knowing the type of `id` in L38